packages/api-client/tests/test-collections.js

```diff
@@ -19,7 +19,7 @@ test('deleteCollection calls the callback with the expected object', async (t) =
     payload: {
       httpMethod: 'DELETE',
       resource: '/{proxy+}',
-      path: `/collections/${t.context.collectionName}/${encodeURIComponent(t.context.collectionVersion)}`,
+      path: `/collections/${encodeURIComponent(t.context.collectionName)}/${encodeURIComponent(t.context.collectionVersion)}`,
```

Comment:
Do we need tests (maybe not here) that assert the actual encoding works and is able to decode the params? Perhaps not if that logic is buried in one of our packages. Just want to be sure it's tested somewhere by someone.

Reply:
So... I would assert that this follows the pattern of the other endpoints. But not much more than that. I found nothing dealing with testing that decoding worked properly in these unit tests. But in the integration test I modified, the collectionVersion has a `/`, so I believe that was testing the encoding and decoding worked, and I _think_ that is where the verification that the encoding and decoding of the new values with `/`s is also happening. (maybe that's where the 'well tested' can be asserted?)
I am definitely not opposed to adding more or better testing. But I don't know where to do that, and I'm not sure what type of tests to add.

Reply:
But you are confident this meets the requirements? I'm ok with the tests here matching the others if YOU have enough confidence that this is working as expected. In cases like this I tend to either add additional tests OR do some ad-hoc testing to ensure the user gets what they expect. If additional automated tests don't make sense because there isn't a good spot for them and this feature is mostly handled by library functions, etc. that's fine, so long as we can confidently assert that this does what the ticket calls for.

Reply:
Discussed in a huddle. The existing tests have been updated with the use case in the ticket's AC. Modifying those tests according to the ticket's AC led to successful reproduction of the failure, adding the encoding fixed the errors. I'd call this successful testing of the AC.